src/util/seasonNumber.ts

```diff
@@ -0,0 +1,11 @@
+let seasonNumber = 1;
+
+export default {
+  getSeasonNumber(): number {
+    return seasonNumber;
+  },
+
+  setSeasonNumber(newSeasonNumber: number): void {
+    seasonNumber = newSeasonNumber;
+  },
+};
```

Comment:
This stuff will likely need to be in the database

Reply:
OK, I will consider it.

Reply:
You can have a look at lastIds for an example :). This is a model that only ever has one entry in it and is incremented for forum use.